packages/web/src/types.ts

```diff
@@ -236,3 +236,15 @@ export type AppCallsParam = BasicParams & AppOptionalFlags & {
 export type ASADef = z.infer<typeof ASADefSchema>;
 
 export type ASADefs = z.infer<typeof ASADefsSchema>;
+
+export interface RequestError extends Error {
+  response?: {
+    statusCode: number
+    text: string
+    body: {
+      message: string
+    }
+    error?: Error
+  }
+  error?: Error
+}
```

Comment:
I think we have a similar type defined in `algob/types`. Could you please check? We can use this one as common.

Reply:
no we don't have it. It was present in types-algosdk. i couldn't find it in js-sdk